docs/pages/config-reference.mdx

```diff
@@ -298,6 +298,10 @@ auth_service:
     # the configured `data_dir` .
     license_file: /var/lib/teleport/license.pem
 
+    # Send a custom message to the client when they are disconnected due to
+    # inactivity. (Currently only supported for SSH connections)
```

Comment:
This is great thank you! One small request, do you mind also adding `The empty string indicates that no timeout message will be sent`?